source/unified-test-format/unified-test-format.rst

```diff
@@ -3752,6 +3760,9 @@ spec changes developed in parallel or during the same release cycle.
 Change Log
 ==========
 
+:2022-09-02: **Schema version 1.11.**
```

Comment:
Also make sure to bump the spec version and last modified date at the top.

Reply:
done